main.cpp

```diff
@@ -209,28 +211,25 @@ void ButtonFactory::designButton()
 
 void ButtonFactory::shipButtons(std::string address, int buttonType, int numToShip)
 {
-    address = "2904 Lark St. Moraine, ND 54206";
-    buttonType = 45;
-    numToShip = 140;
+    std::cout << std::endl << "Ship To: " << address << std::endl << "Type: " << buttonType << std::endl << "Quantity: " << numToShip << std::endl;
 }
 
 void ButtonFactory::Button::orderButton(std::string sku, int amount)
-{
-    sku = "HG74956";
-    amount = 23;
+{   
+    amount -= stockQuantity;
+    skuInfo = sku;
 }
 
 int ButtonFactory::Button::getRadius(std::string sku)
 {
-    sku = "BK203-UT";
-    return 30;
+    skuInfo = sku;
+    return radiusInMm;
 }
 
 std::string ButtonFactory::Button::getButtonColor( std::string sku )
 {
     sku = ButtonFactory::Button::skuInfo;
```

Comment:
you do NOT need to fully qualify the name of the member variable inside member functions in order to access its value.
- [ ] revise